py-polars/docs/source/reference/expressions/boolean.rst

```diff
@@ -16,8 +16,8 @@ Boolean
     Expr.is_infinite
     Expr.is_last
     Expr.is_nan
-    Expr.is_not
```

Comment:
We have to keep the documentation of the deprecated function.

Reply:
Thanks, will do.